schema/artist/index.js

```diff
@@ -290,6 +290,9 @@ const ArtistType = new GraphQLObjectType({
           active: {
             type: GraphQLBoolean,
           },
+          status: {
+            type: GraphQLString,
```

Comment:
There's an EnumType for this: https://github.com/artsy/metaphysics/blob/accac9516e6b3410bf10793929b93c2b4fe6805c/schema/input_fields/event_status.js